### ISSUE TITLE
Bug? SEPA checkbox in checkout page not visible although activated

### ISSUE THREAD

**Issue by s-a-s-k-i-a** [vendidero/woocommerce-germanized#61]
Hey there!
Not sure why, but somehow the gzd css for display:none; is being applied to the SEPA checkbox in checkout page, although I activated the checkbox in checkout in SEPA gateway settings.

the checkbox remains invisible even when SEPA has been chosen from available payment methods in checkout.

any thoughts on why?
Site where this happens: https://ckvoicelessons.de/warenkorb/?add-to-cart=7033

EDIT: it also happen on this site: https://www.erlebniskochen.de/checkout/

(latest Gzd version, latest WooCommerce, latest WP)

**Comment by s-a-s-k-i-a** [comment]
Ach Mensch ich Dösel... da ist ein jQuery verantwortlich.. das ist quasi ein "Feature". Solange die SEPA Lastschrift-Felder nicht vollständig ausgefüllt sind, ist diese Checkbox ausgeblendet. 
Habs gerade in der Datei direct-debit.js im Germanized Plugin gefunden. 

Anstatt einfach mal was in die Felder einzutragen... aber wer soll da schon draufkommen? 🗡 
Na also ich denke, man kann diesen "Bugreport" schließen. 
Es funktioniert wie gewollt.

English:
I just noticed that there is jQuery code responsible for this behaviour. I just found the file direct-debit.js in Germanized plugin files. In that file it states that as long as the Direct debit input fields in payment method section of checkout are empty/not filled out, this checkbox remains invisible.

So everything works as intended.  Just a bit confusing
